docs/docs/general/06-Tutorials/zip-images-upload-googledrive-windows.md

```diff
@@ -9,8 +9,11 @@ This tutorial will show you how to zip files on your file system and then upload
 <div class="cookieconsent-optout-marketing">
   Please <a href="javascript:Cookiebot.renew()">accept marketing-cookies</a> to watch this video.
 </div>
+<div class="cookieconsent-optin-marketing">
+  <a href="javascript:rerunCookieConsentScripts(0)">Please click to watch this video.</a>
```

Comment:
Can you remove the retry parameter? (You can still do a retry but the consumer does not really need to care.)